src/lists/views/search-result-list-template.html

```diff
@@ -9,37 +9,39 @@
   -->
 
 <div ng-show="options.obibaListOptions.searchForm">
-    <list-search-widget type="type"></list-search-widget>
-</div>
-<div ng-show="display === 'list'">
-    <div class="row voffset3">
-        <div class="col-md-12">
-            <span role="presentation" ng-repeat="res in resultTabsOrder"
-                  ng-class="{active: activeTarget[targetTypeMap[res]].active && resultTabsOrder.length > 1, disabled: resultTabsOrder.length === 1}"
-                  ng-if="options[targetTypeMap[res]].showSearchTab">
+    <div class="row">
+        <div class="col-md-2"><span role="presentation" ng-repeat="res in resultTabsOrder"
+                                     ng-class="{active: activeTarget[targetTypeMap[res]].active && resultTabsOrder.length > 1, disabled: resultTabsOrder.length === 1}"
+                                     ng-if="options[targetTypeMap[res]].showSearchTab">
```

Comment:
I totally disapprove these changes. Result is not good at all for pages other than the studies one.